executor/chroot/chroot_executor_test.go

```diff
@@ -31,13 +31,14 @@ func Test(t *testing.T) {
 					},
 				},
 			}
-			executor := &Executor{
+			e := &Executor{
 				workspacePath: "chroot_workspace",
 			}
-			So(os.Mkdir(executor.workspacePath, 0755), ShouldBeNil)
+			So(os.Mkdir(e.workspacePath, 0755), ShouldBeNil)
 
 			Convey("We should get an InputError", func() {
-				So(func() { executor.Run(formula) }, testutil.ShouldPanicWith, input.Error)
+				result := e.Start(formula).Wait()
+				So(result.Error, testutil.ShouldBeErrorClass, tar.Error)
```

Comment:
This check can stay against `input.Error` -- ShouldBeErrorClass checks the hierarchy.  Future work should swim towards a test like this not importing any specific input impl so we can move it towards tables of tests.